include/libtorrent/alert_types.hpp

```diff
@@ -117,10 +117,10 @@ namespace libtorrent
 		virtual std::string message() const override;
 
 		// The peer's IP address and port.
-		tcp::endpoint ip;
+		tcp::endpoint const ip;
```

Comment:
the members that you made const, is there a reason behind picking just those? I imagine if alert members should be const, they should all be const. or what do you think?